package/kernel/ath10k-ct/Makefile

```diff
@@ -8,14 +8,14 @@ PKG_LICENSE_FILES:=
 
 PKG_SOURCE_URL:=https://github.com/greearb/ath10k-ct.git
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_DATE:=2021-09-22
-PKG_SOURCE_VERSION:=e6a7d5b5b834737cd12e357b5efdc2e42d923bf6
-PKG_MIRROR_HASH:=62a1b97089d3561730656ef73beb3cd77231ec636645115cc1bbb3c6c84a6fe3
+PKG_SOURCE_DATE:=2021-11-21
+PKG_SOURCE_VERSION:=e8e166fee23226e5d495afbdb7ed681d747d115b
+PKG_MIRROR_HASH:=f455df42e42c49381cbc3b8e5b510d877e2f9974641cccb73408f0c5b96c34ce
```

Comment:
This hash looks wrong to me, I am getting this:
```
Hash of the local file ath10k-ct-2021-11-21-e8e166fe.tar.xz does not match (file: 771193292242fd8c198e9e379aa90882b1f5a144fbd77e09fbc97aca8ffe84e1, requested: f455df42e42c49381cbc3b8e5b510d877e2f9974641cccb73408f0c5b96c34ce) 
```

Reply:
will update but strange i extracted the hash from the same warning when i updated the package...